### PR TITLE
[Enhancement] Support persist external table analyze status

### PR DESCRIPTION
**Pull request by Youngwb**
#23327
Support persist external table analyze status

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [x] Yes, this PR will result in a change in behavior.
- [ ] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

:disappointed: **fail** : 11 / 41 (26.83%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeMgr.java|0|7|00.00%|[186, 187, 188, 189, 598, 599, 640]|
|:large_blue_circle:|com/starrocks/statistic/ExternalAnalyzeStatus.java|0|5|00.00%|[218, 219, 220, 223, 224]|
|:large_blue_circle:|com/starrocks/persist/EditLog.java|0|8|00.00%|[919, 920, 921, 924, 925, 926, 1893, 1901]|
|:large_blue_circle:|com/starrocks/journal/JournalEntity.java|0|6|00.00%|[896, 897, 898, 901, 902, 903]|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|4|8|50.00%|[1042, 1046, 1126, 1127]|
|:large_blue_circle:|com/starrocks/statistic/StatisticExecutor.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/gson/GsonUtils.java|5|5|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

:heart_eyes: **pass** : 37 / 41 (90.24%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|4|8|50.00%|[1038, 1042, 1122, 1123]|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeMgr.java|7|7|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/ExternalAnalyzeStatus.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticExecutor.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/gson/GsonUtils.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/EditLog.java|8|8|100.00%|[]|
|:large_blue_circle:|com/starrocks/journal/JournalEntity.java|6|6|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java [hunk 1]
```diff
@@ -403,6 +403,10 @@ public class OperationType {
     // constraint properties
     public static final short OP_MODIFY_TABLE_CONSTRAINT_PROPERTY = 11130;
 
+    // external table analyze
```
[the remark this answers is not in the document]
Comment (Youngwb) [reply fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java:411]: I will add other op code later, it's not enough

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

:heart_eyes: **pass** : 37 / 41 (90.24%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|4|8|50.00%|[1038, 1042, 1122, 1123]|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeMgr.java|7|7|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/ExternalAnalyzeStatus.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticExecutor.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/gson/GsonUtils.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/persist/EditLog.java|8|8|100.00%|[]|
|:large_blue_circle:|com/starrocks/journal/JournalEntity.java|6|6|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by gengjun-git — APPROVED** [review]

**Review by stephen-shelby — APPROVED** [review]

**Pull request merged**